src/osd/modules/sound/sdl_sound.cpp

```diff
@@ -304,7 +304,7 @@ void sound_sdl::update_audio_stream(bool is_throttled, const int16_t *buffer, in
 
 	if (stream_buffer->free_size() < bytes_this_frame) {
 		if (LOG_SOUND)
-			fprintf(sound_log, "Overflow: DS=%lu FS=%lu BTF=%lu\n", data_size, free_size, bytes_this_frame);
```

Comment:
These changes break SDL Windows builds - the Windows C runtime lacks the SUS extension %z.

Reply:
Fixed this by switching to streams.